src/components/charts/BorderShutdown.js

```diff
@@ -0,0 +1,208 @@
+import React from "react"
+import Grid from "@material-ui/core/Grid"
+import Paper from "@material-ui/core/Paper"
+import Typography from "@material-ui/core/Typography"
+import outboundIcon from "@components/icons/outbound.png"
+import inboundIcon from "@components/icons/inbound.png"
+import { getCountryFromISO } from "@/utils/mapBaiduCountry"
+import { withStyles } from "@material-ui/core/styles"
+import Tooltip from "@material-ui/core/Tooltip"
+import styled from "styled-components"
+import { useTranslation } from "react-i18next"
+
+const StyledTooltip = withStyles({
+  tooltip: {
+    backgroundColor: "white",
+    color: "rgba(0, 0, 0, 0.87)",
+    fontSize: 16,
+    borderRadius: 14,
+    padding: 10,
+  },
+})(Tooltip)
+
+const Chip = styled(Paper)`
+  color: white;
+  width: auto;
+  height: auto;
+  border-radius: 17pt;
+  cursor: pointer;
+  box-shadow: none;
+  background-color: rgba(255, 255, 255, 0.12);
+  padding: 5px 10px;
+  &:hover {
+    background-color: rgba(255, 255, 255, 0.9);
+    color: black;
+  }
+`
+
+const BorderShutdown = props => {
+  const { data } = props
+  const { t, i18n } = useTranslation()
+
```

Comment:
There are 4 .map loop are doing the same thing 4 times.
Instead we can just compute the data once and shape it just the way we need for rendering.
```suggestion
  const countryNodes = data.map(d => ({
    key: d.node.iso_code,
    country: getCountryFromISO(Number(d.node.iso_code)),
    country_name: withLanguage(i18n, node, 'country'),
    detail: withLanguage(i18n, node, 'detail'),
    status: d.node.status,
  }))
  const country_grouped = _groupBy(countryNodes, 'status')
  
```

Reply:
For the groupBy it meant to be `lodash/groupBy`